frappe/public/js/frappe/form/controls/link.js

```diff
@@ -171,22 +171,24 @@ frappe.ui.form.ControlLink = frappe.ui.form.ControlData.extend({
 						let filter_string = [];
 						for (let [key, value] of Object.entries(args.filters)) {
 
-							if (Array.isArray(value)){
-								filter_string.push(frappe.model.unscrub(key) + " "+value[0]+" "+value[1]);
-							} else {
-								filter_string.push(frappe.model.unscrub(key) + " as " + value);
+							if (Array.isArray(value) && value[1]){
+								filter_string.push(`<b>${frappe.model.unscrub(key)}</b> ${value[0]} <b>${value[1]}</b>`);
+							} else if (value) {
+								filter_string.push(`<b>${frappe.model.unscrub(key)}</b> as <b>${value}</b>`);
 							}
 						}
 
-						filter_string = "Filters applied for " + filter_string.join(", ");
+						if (filter_string.length > 0) {
+							filter_string = "Filters applied for " + filter_string.join(", ");
 
-						r.results.push({
-							label: "<span class='text-muted disable-select' style='line-height: 15px;'>"
-								+ __("{0}", [filter_string])
-								+ "</span>",
-							value: "",
-							action: undefined
-						});
+							r.results.push({
+								label: "<span class='text-muted disable-select' style='line-height: 20px;'>"
+									+ __("{0}", [filter_string])
+									+ "</span>",
+								value: "",
+								action: undefined
```

Comment:
@deepeshgarg007 set this as `null` or remove it.
Setting `null` means the variable has been defined and the developer explicitly wants it to have an empty value (which is what you are trying to do here). Setting a value for a variable as `undefined` is not the idiomatic way of doing it and should be avoided.